radio/src/gui/480x272/splash.cpp

```diff
@@ -19,13 +19,36 @@
  */
 
 #include "opentx.h"
-#include "bmp_splash.lbm"
 
 #if defined(SPLASH)
+
+#include "bmp_splash.lbm"
+
 void drawSplash()
 {
-  lcd->clear();
-  lcd->drawBitmap((LCD_W-BMP_SPLASH.getWidth())/2, (LCD_H-BMP_SPLASH.getHeight())/2, &BMP_SPLASH);
+    static bool     loadImgFromSD  = true;
+    static BitmapBuffer* splashImg = NULL;
+
+    if (loadImgFromSD && splashImg == NULL) {
+        bool sd_mounted = sdMounted();
```

Comment:
is it possible to go there? if yes would you add a comment?

Reply:
It definitely is, that's what happens the very first time the splash function is called. The goal is to execute this code only once, whether it failed or not, we should not re-try later.

Reply:
But normally, the SD card should be unmounted when splash() is called the first time. So basically the case where the SD is already mounted is only handled here as a safeguard.